gradle-plugins/compose/build.gradle.kts

```diff
@@ -77,7 +75,9 @@ val shadow = tasks.named<ShadowJar>("shadowJar") {
     val fromPackage = "de.undercouch"
     val toPackage = "org.jetbrains.compose.$fromPackage"
     relocate(fromPackage, toPackage)
-    archiveClassifier.set("shadow")
+    archiveBaseName.set("shadow")
+    archiveClassifier.set("")
```

Comment:
archiveClassifier changes from "shadow" to empty string ""
Is it correct?

Reply:
Yes, for some reason, the new version requires to remove the classifier